webui/react/src/components/Metadata.module.scss

```diff
@@ -1,5 +1,3 @@
 .base {
-  :global(.ant-tree-node-content-wrapper-close) {
-    font-weight: bold;
-  }
+  margin-bottom: 32px;
```

Comment:
nit: should use the spacing variables. 32px is also pretty large, i suggest `var(--spacing-xl-2)` which is 20px

Reply:
fixed